dags/services/AWSService.py

```diff
@@ -0,0 +1,40 @@
+import boto3
+from airflow.models import Variable
+
+# AWS creds
+AWS_CREDS = {
+    "AWS_ACCESS_KEY_ID": Variable.get("TOWER_DB_ACCESS_KEY"),
+    "AWS_SECRET_ACCESS_KEY": Variable.get("TOWER_DB_SECRET_ACCESS_KEY"),
+}
+# AWS region
```

Comment:
Just a comment here, should we be using https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/operators/s3.html?